src/kerberos/mod.rs

```diff
@@ -873,9 +885,9 @@ impl<'a> Kerberos {
                 // the original flag is
                 // GSS_C_MUTUAL_FLAG | GSS_C_REPLAY_FLAG | GSS_C_SEQUENCE_FLAG | GSS_C_CONF_FLAG | GSS_C_INTEG_FLAG
                 // we want to be able to turn of sign and seal, so we leave confidentiality and integrity flags out
-                let flags = builder.context_requirements.into();
-                let mut checksum_value = ChecksumValues::default();
-                checksum_value.set_flags(flags);
+                // let flags = builder.context_requirements.into();
+                let checksum_value = ChecksumValues::default();
+                // checksum_value.set_flags(flags);
```

Comment:
We have a problem here. During the dev testing, I had a problem with this code. When I uncomment these `flags`, then the CredSSP server returns an `IvalidToken` error. I'm not sure why, but when commented on it, then authentication succeeded.

Reply:
For now, can you leave a FIXME at this place? We’ll have to address this before merging into master.

Reply:
I added a comment here.
Can I take this issue today? I want to merge smart card implementation into a master quickly.